### ISSUE TITLE
Background and foreground colors do not have a sufficient contrast ratio with transparency.

### ISSUE THREAD

**Issue by vixducis** [GoogleChrome/lighthouse#7301]
Steps to reproduce
1. Run he accessibility check of LH on https://www.scoutswechel.be/

Current behaviour:
Following error is thrown:
Background and foreground colors do not have a sufficient contrast ratio
This happens for following type of elements:
![walter](https://user-images.githubusercontent.com/378998/53242633-96432f80-36a5-11e9-9bdf-b8e43dfecdb4.png)

Expected behaviour: 
The contrast should be sufficient to qualify for WCAG AA. The resulting foreground color is #CEE0E2, the background color is #20647B, this should result in a contrast of 4.86:1. 
Not sure how lighthouse this this, but the colors are a result of 2 layers of transparancy: rgba(255,255,2550.8) on rgba(0,0,0,0.5). Maybe this results in an issue?

I checked this with axe as well and while other elements are failing, the one that I mentioned does not pop up in axe.

* Lighthouse version: 3.2.0
* Operating System: Windows 8..1

**Comment by patrickhulce** [comment]
We directly running axe. In latest version of Lighthouse and latest version of axe I see the same nodes being reported for this page, including the one you've flagged here. If you believe this behavior is incorrect, please file the issue directly with axe. https://github.com/dequelabs/axe-core/issues

![image](https://user-images.githubusercontent.com/2301202/53258829-72222700-3682-11e9-9350-cf24fe4488b2.png)
